frontend/frontend.js

```diff
@@ -0,0 +1,116 @@
+            h2.innerHTML = `${name} was added to ${className}`;
+            ul.appendChild(h2);
+        }
+
+    const printStudents = async (className, city, failing) => {
+        //if city is blank it is === ""
+        //if failing is not clicked false or other wise
+        let list = await axios.get(`http://localhost:3000/class/${className}/students`);
+        ul.innerHTML = "";
+        if(city != "" && failing === true){
+            let h2 = document.createElement("h2");
+            h2.innerHTML = `${className} students from ${city} and failing`;
+            ul.appendChild(h2);
+            for(let i = 0; i < list.data.length; i ++){
+                if(list.data[i]["city"] === city && list.data[i]["grade"] < 70){
+                    let li = document.createElement("li");
+                    li.innerHTML = `Name: ${list.data[i]["name"]} Age: ${list.data[i]["age"]} City: ${list.data[i]["city"]} Grade: ${list.data[i]["grade"]}`;
+                    ul.appendChild(li);
+                }
+            }
+        } else if(city != "" && failing === false){
+            let h2 = document.createElement("h2");
+            h2.innerHTML = `${className} students from ${city}`;
+            ul.appendChild(h2);
+            for(let i = 0; i < list.data.length; i ++){
+                if(list.data[i]["city"] === city){
+                    let li = document.createElement("li");
+                    li.innerHTML = `Name: ${list.data[i]["name"]} Age: ${list.data[i]["age"]} City: ${list.data[i]["city"]} Grade: ${list.data[i]["grade"]}`;
+                    ul.appendChild(li);
+                }
+            }
+        } else if(city === "" && failing === true){
+            let h2 = document.createElement("h2");
+            ul.appendChild(h2);
+            h2.innerHTML = `Failing ${className} students`;
+            for(let i = 0; i < list.data.length; i ++){
+                if(list.data[i]["grade"] < 70){
+                    let li = document.createElement("li");
+                    li.innerHTML = `Name: ${list.data[i]["name"]} Age: ${list.data[i]["age"]} City: ${list.data[i]["city"]} Grade: ${list.data[i]["grade"]}`;
+                    ul.appendChild(li);
+                }
+            }
+        } else {
+            let h2 = document.createElement("h2");
+            h2.innerHTML = `${className} students`;
+            ul.appendChild(h2);
+            for(let i = 0; i < list.data.length; i ++){
+                let li = document.createElement("li");
+                li.innerHTML = `Name: ${list.data[i]["name"]} Age: ${list.data[i]["age"]} City: ${list.data[i]["city"]} Grade: ${list.data[i]["grade"]}`;
+                ul.appendChild(li);
+            }
+        }
+    }
+
+    //Adds a class and puts it in the backend
+    form1.addEventListener("submit", async (e) => {
+        e.preventDefault();
+        addClass(addClassName.value, addTeacher.value);
+        window.alert('The class was added');
```

Comment:
In all these functions we want to display the response we are getting from the backend